guides/promotions/promotion-rules.md

```diff
@@ -0,0 +1,111 @@
+# Promotion rules
+
+The `Spree::PromotionRule` model sets a rule that determines whether a promotion
+is eligible to be applied. Promotions may have no rules or many different rules.
+
+An example of a typical promotion rule would be a minimum order total of $75
+USD or that a specific product is in the cart at checkout.
+
+Many conventional types of promotion rules are included by default. Multiple
+promotion rules can exist on each promotion, although no rules are required.
+
+# Available promotion rule types
+
+The following classes are [subclasses of the `Spree::Promotion::Rules`
+model][promotion-rules]:
+
+- `FirstOrder`: Eligible for a customer's first order only.
+- `FirstRepeatPurchaseSince`: Eligible for a customer's first repeat purchase
+  since a specified date.
+- `NthOrder`: Eligible for a customer's *n*th order only.
+- `ItemTotal`: Eligible if the order total is less than or greater than a
```

Comment:
I would try to be more specific that it's the order's item total. (i.e.: Before taxes/shipping/other discounts/etc.)